### ISSUE TITLE
Links widget must have null alt attributes on all images (so it reads out the destination rather than the alternative test)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/uofg-components#20]
This fails WCAG 2.0 AA if we're reading out the alternative text along with the destination
<img width="1523" height="764" alt="Image" src="https://github.com/user-attachments/assets/32fc4f2d-e1bf-4f39-8cc5-138b8dd7086b" />
